Makefile

```diff
@@ -100,8 +100,8 @@ clean:
 
 .PHONY: \
 	all \
-	tools \
-	man \
+	$(TOOLS) \
+	$(MAN) \
```

Comment:
You don't want to make this change.  The [`tools` target][1] is phony (building it does not create a file called `tools`), but the [`$(TOOLS)` targets][2] are real (building them creates the expected `oci-*` files).  Same for `man` (phony) and `$(MAN)` (not phony).

[1]: https://github.com/opencontainers/image-tools/blob/a358e03fde4e3628bf9fb7656bf643b63f975636/Makefile#L32
[2]: https://github.com/opencontainers/image-tools/blob/a358e03fde4e3628bf9fb7656bf643b63f975636/Makefile#L38-L39